### ISSUE TITLE
*Starting Out with DEPNotify...

### ISSUE THREAD

**Issue by Tech619** [hoppyshaw/DEPnotify-WS1#1]
Not an issue, but I'm just starting out with DEPNotify and have zero idea on what to do. Is there a step-by-step guide for total newbies to follow? Like taking baby-steps, so to speak?

**Comment by hoppyshaw** [comment]
Hi Tech619. What mdm are you looking to use it with? That way, I can point you to a couple different guides. In the meantime. the best thing you can do is join the macadmins slack channel if not already a member. Lots of GREAT info there along with folks using JAMF, WS1, and other MDM's. That's where I would go to find links based on what your specific mdm is. The guide I started with is on the devs gitlab page here: https://gitlab.com/Mactroll/DEPNotify   

Here is the link to join the depnotify group on macadmins slack channel: https://macadmins.slack.com/archives/C523HLQ83

**Comment by Tech619** [comment]
Hi Hoppy,

I'm Bhupinder. Nice to meet you. Thanks for the quick reply.

I saw your YouTube video from MacAdmin conference and it gave me a huge
amount of info on what you did and how you accomplished your process using
DEPNotify. The screenshots of the folder structure and your emphasis on
changing the permissions *before creating the DEPNotify package was
especially helpful.

I also visited the links you pointed out and downloaded the latest version
of DEPNotify. I also joined MacAdmins on Slack and a few other channels.

I'll read up some more on it and try to get a picture of the process . I'm
using JAMF as an MDM. The thing is; the more I read, the more it confused
me so I reached out to you.


Like you said, you aren't a huge scripting person. But your definition of
not being a scripting person is obviously different than mine. I wouldn't
know what I'm looking at unless it's in plain English. I mean imagine
Grandma trying to understand Python or BASH Script. Yeah. I'm THAT
unfamiliar with scripting. * I did come across Scripting 101 and 102 from
Talking Moose on YouTube.

I'll let you know how things go.

Again thanks for the quick response.

You rock!
